File: week8/ch.Choe/Week8/Tea.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Week8
{
    class Tea : CaffeBeverage
    {
        protected override void Add()
        {
            Console.WriteLine("Tea Add");
        }

        protected override void Brew()
        {
            Console.WriteLine("Tea Brew");
        }
    }
}

File: week8/sgmsgood/Tea.java
package Templete;

public class Tea extends CaffeineBeverage {
	@Override
	void brew() {
		System.out.println("차를 우려내는 중");
	}

	@Override
	void addCondiments() {
		System.out.println("레몬을 추가하는 중");
	}
}
